lib/utils/index.js

```diff
@@ -2527,6 +2528,23 @@ function getVueComponentDefinitionType(node) {
       if (callee.name === 'createApp') {
         // for Vue.js 3.x
         // createApp({})
+
+        const variable = findVariable(context.getScope(), callee)
+
+        // only lint the createApp function that import from vue
+        if (variable !== null && variable.defs.length === 1) {
+          const def = variable.defs[0]
+          if (
+            def.type === 'ImportBinding' &&
+            def.node.type === 'ImportSpecifier' &&
+            def.node.imported.type === 'Identifier' &&
+            def.node.parent.type === 'ImportDeclaration' &&
+            def.node.parent.source.value !== 'vue'
```

Comment:
`@vue/composition-api` should also be allowed; please also add test cases for that if there aren't any yet.

Reply:
And maybe also add a test with `createApp` defined in the same file instead of imported from another file?